command/alias.go

```diff
@@ -112,3 +114,53 @@ func processArgs(args []string) []string {
 
 	return newArgs
 }
+
+var aliasListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List your aliases",
+	Long:  `This command prints out all of the aliases gh is configured to use.`,
+	Example: `$ gh alias list
+co:   pr checkout
+bugs: issue list --label="bugs"`,
```

Comment:
I would offer here that, in general, we don't need to provide examples of invocations that take no arguments. The `Usage` line already covers that